demo/jenkinsfile-runner/kubernetes.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: jenkinsfile-runner
+spec:
+  template:
+    spec:
+      containers:
+      - name: jenkinsfile-runner
+        image: onenashev/cwp-jenkinsfile-runner-demo
```

Comment:
Same image as in the demo, just renamed. Maybe we need to rename it across the demo to avoid confusion